proto/commonfate/control/config/v1alpha1/aws_rds_database_user.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+package commonfate.control.config.v1alpha1;
+
+import "commonfate/authz/v1alpha1/read_only.proto";
+
+service AWSRDSDatabaseUserService {
```

Comment:
Could you please add a comment here on the service as to why this is different from `AWSRDSDatabaseService` and when you'd want to use each one? (might also be worth a comment on `AWSRDSDatabaseService` too)

Reply:
Ahh right, these actually need to be removed. They were feom the demo last week